### DISCUSSION TITLE
Ezviz con Hassio

### DISCUSSION THREAD

**Discussion by dcamb** [indomus/forum#203] [Hardware & Firmware]
Ciao a tutti
qualcuno sa dirmi se è stata ripristinata l'integrazione cloud delle ipcam ezviz?
Grazie ciao Davide

**Comment by snakuzzo** [comment]
https://www.home-assistant.io/integrations/ezviz/

**Reply by dcamb** [reply]
Non so se sia attendibile ma in alcuni articoli si dice che l'integrazione è stata eliminata
**LINK RIMOSSO**
effettivamente io non riesco a farla funzionare...

**Reply by snakuzzo** [reply]
NON SONO AMMESSI LINK A SITI TERZI. Leggi il [REGOLAMENTO](https://github.com/indomus/forum)

**Reply by dcamb** [reply]
Sorry, 
ora lo so :-)
sto cercando i log per vedere se ci sono degli errori...

**Reply by snakuzzo** [reply]
L'integrazione per Ezviz dovrebbe essere in dev...dovresti trovarla nella prossima release di HA

https://github.com/home-assistant/core/pull/45722

**Reply by dcamb** [reply]
Grazie, 
nel frattempo sto testando un componente di integrazione da HACS,
vediamo se funziona.
Questo è l'errore:
![Cattura](https://user-images.githubusercontent.com/81462996/115925561-a1f73000-a481-11eb-9e6d-ab535cf73ce4.JPG)
